src/translations/en.json

```diff
@@ -23,5 +23,12 @@
     "content2": "Be the first to score 21 points in this explosive battle of wits. Mount a valient offense while disupting your opponent with dastardly tricks. Do you have what it takes to become the Lord of the Deep?",
     "readRules": "Read the Rules",
     "joinDiscord": "Join our discord community"
+  },
+  "lobby":{
```

Comment:
Question: Is this verbiage used else where in the application? To avoid duplication, maybe it would be better to keep them at the global level instead of scoped only for the lobby.

Reply:
@dan-sarich good question. All the text that's been put under "lobby" here is specific to the lobby and does not appear elsewhere. The text that is shared across pages in the app e.g. the names of pages in the nav links are under the "global" section above. 

The lobby has very limited text overall and it happens not to have any significant overlap with text in other pages so everything here is specific to the lobby